src/Library/demos/Screenshot/main.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Screenshot",
+  "category": "portal",
+  "description": "Keep a snapshot of your current screen",
+  "panels": [
+    "ui",
```

```suggestion
    "code",
```

it's more important here